src/Proxy/User.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace DanielBadura\Redmine\Api\Proxy;
+
+class User 
+{
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @param $id
+     * @param $name
+     */
+    function __construct($id, $name)
+    {
+        $this->id   = $id;
+        $this->name = $name;
+    }
+
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()

```

Comment:
i justed checked with the api, there actually should be a name attribute. But I our user struct does not take this value at the moment, so when i dumped an issue there is no name.